crates/bevy_gizmos/src/circles.rs

```diff
@@ -0,0 +1,142 @@
+//! Additional [`Gizmo`] Functions -- Arrows
+
+use crate::prelude::Gizmos;
+use bevy_math::{Quat, Vec2, Vec3};
+use bevy_render::color::Color;
+use std::f32::consts::TAU;
+
+pub const DEFAULT_CIRCLE_SEGMENTS: usize = 32;
+
+pub fn circle_inner(radius: f32, segments: usize) -> impl Iterator<Item = Vec2> {
```

Comment:
This was made pub, but should probably stay private.

Reply:
If you'd prefer it to be pub, it needs a better name and doc strings :)

Reply:
Yeah i thought it was pub before. just reverted it; that can be a different PR.

Reply:
Heck i thought this was about DEFAULT_CIRCLE_SEGMENTS

Reply:
not sure why i made this public. seems to work fine when made private.
maybe I made it pub when some of the stuff I moved was still in gizmos.rs